### ISSUE TITLE
Search for multiple tags in DHTML version

### ISSUE THREAD

**Issue by dirkrombauts** [picklesdoc/pickles#283]
Currently the DHTML version allows to search for only one tag at a time. In larger specifications, there are often many tags used in specific combinations. It would be beneficial if we could search for multiple tags.

**Comment by dirkrombauts** [comment]
@albertdabrowski could you please enhance this description? One of the questions I have is:

```
Given I have these scenarios:

@tag1
Scenario: Scenario 1

@tag1 @tag2
Scenario: Scenario 2

@tag 2
Scenario: Scenario 3

When I search for "@tag1 @tag2"
Then the result is ???
```

**Comment by albertdabrowski** [comment]
In our case we would like to have "OR", but I think "AND" would be also valuable. So I think in this case it's a matter of search capability. Do we want to keep it simple or more like for example search in JIRA?

**Comment by dirkrombauts** [comment]
OR would mean that 

```
When I search for "@tag1 @tag2"
Then the result is "Scenario 1, Scenario 2, Scenario 3"
```

Right? I'm happy you said OR, because the OR behaviour is consistent with many other search-by-property search definitions that treat different properties as AND criteria, and different values of the same property as OR criteria.

I have to admit I have no experience with JIRA.

**Comment by dirkrombauts** [comment]
@aaronjrich would you have a moment to get in touch with me? I'd like to ask you a few questions about the DHTML version, with the goal that I am able to extend it myself. I feel bad about having to bother you every time something with the DHTML version comes up.

**Comment by aaronjrich** [comment]
@dirkrombauts, sorry about now getting back sooner.  Let me know what time works for you.

**Comment by dirkrombauts** [comment]
@aaronjrich tomorrow (19 February) in the forenoon would work for me (for example). I'm in GMT+2, I don't know which timezone you are in ...

BTW, if you are on gitter, you can send me a message there, that's perhaps easier than filling up this feature request.

**Comment by aaronjrich** [comment]
Apparently gitter is blocked here where I work.  Is there another way to proceed with this?

**Comment by dirkrombauts** [comment]
You can email me at dirk dot rombauts at picklesdoc dot com . Does Skype work at your workplace?

**Comment by dirkrombauts** [comment]
Released in version [2.6.0](https://github.com/picklesdoc/pickles/releases/tag/v2.6.0)

**Comment by albertdabrowski** [comment]
Great, thanks for the efforts.

Appreciate,
Albert
